packages/components/nodes/vectorstores/Zep/Zep_Upsert.ts

```diff
@@ -20,8 +20,8 @@ class Zep_Upsert_VectorStores implements INode {
     outputs: INodeOutputsValue[]
 
     constructor() {
-        this.label = 'Zep Upsert Document'
-        this.name = 'zepUpsert'
+        this.label = 'Zep Upsert Document - Open Source'
+        this.name = 'Zep Upsert - Open Source'
```

Comment:
prefer to revert all the `name` back to the initial ones, because changing exsiting will likely cause the existing flow not working, and also they need to be `camelCase`

Reply:
@HenryHengZJ Got it. Just reverted name changes + updated the new ones to be camelCase.